tests/privacy/test_privacy_mechanism.py

```diff
@@ -484,3 +485,228 @@ def test_norm_clipping_only(self, order, clipping_bound, expected_arrays,
                                  is_local_privacy=True)
         assert get_overall_value(
             metrics[name]) == pytest.approx(expected_clip_fraction)
+
+    @pytest.mark.parametrize('ops_module', framework_fixtures)
```

Comment:
Since the mechanisms themselves are tested in other unittests, you can simplify this a lot by just using MagicMock as mechanisms input to JointMechanism.
and the test will be much faster, as i recall these statistical tests are not super fast.

Reply:
I've rewritten the test as suggested